vendor/knative.dev/networking/pkg/apis/networking/v1alpha1/ingress_lifecycle.go

```diff
@@ -46,6 +46,9 @@ func (is *IngressStatus) GetCondition(t apis.ConditionType) *apis.Condition {
 // InitializeConditions initializes conditions of an IngressStatus
 func (is *IngressStatus) InitializeConditions() {
 	ingressCondSet.Manage(is).InitializeConditions()
+
+	// Deprecatd, do not set.
```

Comment:
```suggestion
	// Deprecated, do not set.
```

Reply:
That's vendor

Reply:
Sent https://github.com/knative/networking/pull/140 to fix this.